tuts/clone-a-live-chain/v2.0.0-alpha.6/index.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: index
+lang: en
+title: Clone a Live Chain
+---
+
+In this tutorial we will learn to start a Substrate chain that takes its initial state from an existing live blockchain. This technique is useful for investigating bugs, testing runtime upgrades, performing airdrops, and "forking" a community during a contentious disagreement.
```

Comment:
It would be interesting for this tutorial to be wrapped up in an exercise of investigating a bug or testing an upgrade...or both. It would also be nice to hear about _how_ this technique can be used for airdops or forking. Also may want to consider rewording the events that may lead to a fork.